src/util.rs

```diff
@@ -11,3 +14,33 @@ pub(crate) fn to_snake_case<S: AsRef<str>>(ident: &S) -> String {
 
     snake_case
 }
+
+pub(crate) struct UniqueIdentifierIterator {
+    buffer: Vec<u8>,
+}
+
+impl UniqueIdentifierIterator {
+    pub(crate) fn new() -> Self {
+        UniqueIdentifierIterator {
+            buffer: vec!['a' as u8],
+        }
+    }
+}
+
+impl Iterator for UniqueIdentifierIterator {
+    type Item = Ident;
+
+    // Generates inifinite length strings from ASCII chars a-z
```

Comment:
Also, that's a really clever, generic way of making unique identifiers. I'm probably going to steal this in a few places 👍